### ISSUE TITLE
Stopped working with newest version of Youtube

### ISSUE THREAD

**Issue by saabdeep** [pylerSM/YouTubeBackgroundPlayback#29]
The latest YouTube update broke this great module. The app updated to version 10.11.55 on March 27th. I tried updating to the newest version of this module (1.3.7 at the time of this thread) but the notification doesn't even appear when selecting a video as it did before the YouTube update. 

I'm on a rooted HTC One M8 with android version 4.4.4 Kit Kat!

Cheers

**Comment by pylerSM** [comment]
Not gonna to check this since we have newer update 10.12.

**Comment by saabdeep** [comment]
You mean, you won't look into this because I don't have the newest YouTube app version? The latest one I can download from Google Play is 10.11.55, I can't download 10.12:

![image](https://cloud.githubusercontent.com/assets/11758413/6948147/cc04a4d4-d879-11e4-8f7f-3dcd26901915.png)

Now if you are saying that the module works fine in 10.22, point me in the direction of the YouTube apk in that version and I will try to install it and report back. As is, it is broken.

**Comment by pylerSM** [comment]
Google "apkmirror youtube"

**Comment by saabdeep** [comment]
Of course I had already found that mirror, but here is the latest there:

![image](https://cloud.githubusercontent.com/assets/11758413/6948684/63099198-d87d-11e4-98d5-bcaa1e0b4e08.png)

You referenced 10.22, not 10.12.53 which seems to be the latest here. I shall attempt to install 10.12.53 and test!

Cheers

**Comment by pylerSM** [comment]
I never mentioned 10.22.

**Comment by pylerSM** [comment]
You should not install x86 version, install normal arm one.

**Comment by saabdeep** [comment]
So I installed the latest 10.12.53, but there is no change, even after reboot. The HTC One M8 is an extremely popular phone so I'm wondering why others haven't spoken up about this issue..

Thank you for your responsiveness...I just love the feature and was disappointed when it stopped working. No doubt it is Google trying to prevent this since it is a pay-for-"feature" in YouTube now  :-/

**Comment by troidem** [comment]
After installing 10.12 on my HTC One M8 ,Its perfectly working !!

**Comment by saabdeep** [comment]
On you on Lollipop by chance?

**Comment by troidem** [comment]
Yes I am on L Viper Rom

**Comment by saabdeep** [comment]
Ok Good to know. I ended up getting this to work, but I had to uninstall all YouTube updates, then uninstall this module, reboot, re-install both, reboot, then activate the module, reboot. 

After that sequence the module is working :+1: but I do have the same bug as @troidem with the X only removing the notification, but not actually stopping the track from playing. Honestly, this may have been the functionality from past versions but I'm not sure. I can always stop by pausing it from the notification first and then tapping the X to remove the notification. Thanks guys.
